packages/editor/src/components/entities-saved-states/index.js

```diff
@@ -183,12 +186,20 @@ export function EntitiesSavedStatesExtensible( {
 	const [ saveDialogRef, saveDialogProps ] = useDialog( {
 		onClose: () => dismissPanel(),
 	} );
+	const dialogLabel = useInstanceId( EntitiesSavedStatesExtensible, 'label' );
+	const dialogDescription = useInstanceId(
+		EntitiesSavedStatesExtensible,
+		'description'
+	);
 
 	return (
 		<div
 			ref={ saveDialogRef }
 			{ ...saveDialogProps }
 			className="entities-saved-states__panel"
+			role="dialog"
+			aria-labelledby={ dialogLabel }
+			aria-describedby={ dialogDescription }
```

Comment:
We need this to be marked up appropriately, but by adding the role here I think it'll mean [this modal will have two nested dialogs](https://github.com/WordPress/gutenberg/blob/trunk/packages/edit-site/src/components/save-panel/index.js#L124)?